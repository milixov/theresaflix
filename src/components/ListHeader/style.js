import {StyleSheet} from 'react-native';

export const styles = StyleSheet.create({
    container: {
        paddingVertical: 8,
        marginHorizontal: 16,
        fontSize: 18,
        fontWeight: 'bold',
    },
});
