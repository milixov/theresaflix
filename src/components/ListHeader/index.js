import React from 'react';
import {Text} from 'react-native';

import {styles} from './style';

const ListHeader = ({text}) => {
    return <Text style={styles.container}>{text}</Text>;
};

export default ListHeader;
