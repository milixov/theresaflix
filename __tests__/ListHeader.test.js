import React from 'react';
import ListHeader from '../src/components/ListHeader';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
    const tree = renderer.create(<ListHeader />).toJSON();
    expect(tree).toMatchSnapshot();
});
