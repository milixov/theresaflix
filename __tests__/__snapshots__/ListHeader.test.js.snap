// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  style={
    Object {
      "fontSize": 18,
      "fontWeight": "bold",
      "marginHorizontal": 16,
      "paddingVertical": 8,
    }
  }
/>
`;
